independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/RestEasyParamsFilter.java

```diff
@@ -0,0 +1,11 @@
+package org.jboss.resteasy.reactive;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface RestEasyParamsFilter {
```

Comment:
I will defer to @geoand on this PR but my guess is that we need a better name without `RestEasy` in it.